src/patternfly/demos/Card/templates/card-template-status.hbs

```diff
@@ -0,0 +1,143 @@
+            {{/l-flex-item}}
+          {{/l-flex}}
+        {{/l-flex}}
+        {{#> l-flex l-flex--modifier="pf-m-space-items-sm pf-m-nowrap"}}
+          {{#> l-flex-item}}
+            <i class="fas fa-check-circle pf-u-success-color-100" aria-hidden="true"></i>
+          {{/l-flex-item}}
+          {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
+            {{#> l-flex-item}}
+              <a href="#">Image Vulnerabilities</a>
+            {{/l-flex-item}}
+            {{#> l-flex-item}}
+              <span class="pf-u-color-200">0 vulnerabilities</span>
+            {{/l-flex-item}}
+          {{/l-flex}}
+        {{/l-flex}}
+        {{#> l-flex l-flex--modifier="pf-m-space-items-sm pf-m-nowrap"}}
+          {{#> l-flex-item}}
+            <i class="fas fa-check-circle pf-u-success-color-100" aria-hidden="true"></i>
+          {{/l-flex-item}}
+          {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
+            {{#> l-flex-item}}
+              <a href="#">Storage</a>
+            {{/l-flex-item}}
+            {{#> l-flex-item}}
+              <span class="pf-u-color-200">Degraded</span>
+            {{/l-flex-item}}
+          {{/l-flex}}
+        {{/l-flex}}
+        {{#> l-flex l-flex--modifier="pf-m-space-items-sm pf-m-nowrap"}}
+          {{#> l-flex-item}}
+            <i class="fas fa-check-circle pf-u-success-color-100" aria-hidden="true"></i>
+          {{/l-flex-item}}
+          {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
+            {{#> l-flex-item}}
+              <a href="#">Hardware</a>
+            {{/l-flex-item}}
+          {{/l-flex}}
+        {{/l-flex}}
+        {{#> l-flex l-flex--modifier="pf-m-space-items-sm pf-m-nowrap"}}
+          {{#> l-flex-item}}
+            <i class="fas fa-check-circle pf-u-success-color-100" aria-hidden="true"></i>
+          {{/l-flex-item}}
+          {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
+            {{#> l-flex-item}}
+              <a href="#">Insights</a>
+            {{/l-flex-item}}
+          {{/l-flex}}
+        {{/l-flex}}
+      {{/gallery}}
+    {{/card-body}}
+    {{> divider}}
+    {{#> notification-drawer}}
+      {{#> notification-drawer-body}}
+        {{#> notification-drawer-group notification-drawer-group--IsExpanded=card-demo--notication-drawer--IsOpen}}
+          {{#> notification-drawer-group-toggle}}
+            {{#> notification-drawer-group-toggle-title}}
+              {{#> l-flex}}
+                {{#> notification-drawer-group-toggle-title}}
+                  {{#> l-flex l-flex--modifier="pf-m-space-items-sm" l-flex--attribute='style="row-gap: var(--pf-global--spacer--xs);"'}}
+                    {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-md"}}
+                      <span>Notifications</span>
+                    {{/l-flex-item}}
+                    {{#> label label--id="default-red-icon" label--modifier="pf-m-red"}}
```

Comment:
These labels could be in a label group. That's how it's setup in the horizontal grid card demo - https://www.patternfly.org/v4/components/card/html-demos#horizontal-grid-collapsed